### DISCUSSION TITLE
Running Examples Using Docker on JupyterLab

### DISCUSSION THREAD

**Discussion by kylebeni** [NVIDIA-Merlin/Transformers4Rec#758] [Q&A] [answered]
When following tutorial instructions on https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/main/examples, I'm struggling with Step 6 in "Running the Example Notebooks":

> After you log in to JupyterLab, navigate to the /transformers4rec/examples directory to try out the example notebooks.

In the merlin-pytorch docker image, I'm not seeing any folder /transformers4rec/examples. Am I missing a step somewhere (for example, cloning the repository within the docker image)?

This is what I'm seeing when I launch the JupyterLab server:

![image](https://github.com/NVIDIA-Merlin/Transformers4Rec/assets/106107895/b08b7315-9547-46a9-b463-16ae89b76781)

Any help or guidance would be appreciated, thanks.

**Comment by rnyak** [comment, marked as answer]
which platform are you using merlin-pytorch image? The image comes with the examples folder.  you need to first do 
```
cd /transfomers4rec
```

then launch the jupyter lab from there.

**Reply by kylebeni** [reply]
Thank you, that was the step that was missing.
